packages/RatPack.package/RPDynamicEnv.class/class/parsedRequestContentsKey.st

```diff
@@ -0,0 +1,4 @@
+as yet unclassified
+parsedRequestContentsKey 
```

Comment:
A lot of methods in this PR are unclassified. Please categorize them all.

Reply:
pull request :D
Just go to the "Files changed" tab and use your browser's find function to search for "as yet unclassified".

Reply:
For the ContentDeconders on class site are also categories missing, maybe you can find a good Naming.

Reply:
If we don't have namings for these methods yet, I'd call them "constants" or something similar.